fiftyone/core/view.py

```diff
@@ -317,6 +326,7 @@ def summary(self):
             a string summary
         """
         elements = [
+            ("View name:", self.name),
```

Comment:
How about only including `View name` if the view is actually a saved view?

When it is not a saved view, the default name is not meaningful and I think it's helpful to reduce clutter whenever possible on these heavily used outputs.

Reply:
Related suggestion: how about adding a property like `DatasetView.is_saved_view` to allow users to determine if a view is a saved view or not?